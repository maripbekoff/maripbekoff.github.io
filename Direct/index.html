<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>IT - Wikipedia</title>
	<link type="text/css" rel="stylesheet" href="styles/style.css">
	<link rel="icon" href="favicon.ico">
</head>
<body>

	<header>
		<div class="topnav">
			<a class="active" href="index.html">IT</a>
			<a href="guides.html">Guides</a>
			<a href="#about">About</a>
			<a href="https://arnaitours.kz" target="_blank">Sponsors</a>
			<a id="kali" href="https://kali.org/downloads" target="_blank">Kali Linux</a>
		</div>
	</header>
	<div class="bg-image img1"></div>
	<div class="bg-image img2"></div>
	<div class="bg-image img3"></div>

	<div class="bg-text">#maripbekoff</div>
</body>
</html>