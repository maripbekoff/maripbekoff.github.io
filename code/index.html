<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Code Union | Welcome!</title>
	<link rel="stylesheet" type="text/css" href="styles/style.css">
	<link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
</head>
<body>
	<header>
		<h1><a href="index.html">Code Union</a></h1>
		<nav>
			<a href="index.html" class="active">Home</a>
			<a href="#">Guides</a>
			<a href="#">Owner</a>
		</nav>
	</header>
	<hr>
	<section id="wrapper">
		<section id="about">
			<h1 align="center">About</h1>
			<p>Code Union's the site, which was made by <a href="https://vk.com/chingizsck" target="_blank">#maripbekoff.</a>
				 I made it only for practicing my skills on Front-End Development.</p><br>
			<p>But I will also make some guides based on my expirience. For instance it can be "Information Security" or tips and guides of "Web Development".</p>	 
		</section>
		<section id="links">
			<h1 align="center">Links</h1>
			<div class="links">
				Social Network page - <a href="https://vk.com/chingizsck">vk</a><br>
				Github page - <a href="https://github.com/maripbekoff" target="_blank">Github</a><br>
				Kali Linux - <a href="https://kali.org/downloads" target="_blank">Kali</a><br>
			</div>
			<div class="ps">
				<p>If you have ideas how to improve design or other things on this site please <a href="mailto:tech.codeunion@gmail.com">contact me.</a></p>
			</div>
		</section>
	</section>
	<hr>
	<footer>
		<h2>#maripbekoff</h2>
		<p>&copy;Code Union all rights reserved</p>
	</footer>
</body>
</html>