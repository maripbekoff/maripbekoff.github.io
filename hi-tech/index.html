<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <meta charset="utf-8"/>
        <title>HI-TECH | Code Union</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta content="width=device-width, initial-scale=1" name="viewport"/>

        <link href="http://fonts.googleapis.com/css?family=Hind:300,400,500,600,700" rel="stylesheet" type="text/css">
        <link href="vendor/simple-line-icons/simple-line-icons.min.css" rel="stylesheet" type="text/css"/>
        <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>

        
        <link href="css/animate.css" rel="stylesheet">
        <link href="vendor/swiper/css/swiper.min.css" rel="stylesheet" type="text/css"/>

        
        <link href="css/layout.min.css" rel="stylesheet" type="text/css"/>

        
        <link rel="shortcut icon" href="favicon.ico"/>
    </head>
    

    
    <body>

        
        <header class="header navbar-fixed-top">
            
            <nav class="navbar" role="navigation">
                <div class="container">
                    
                    <div class="menu-container">
                        
                        <div class="logo">
                            <a class="logo-wrap" href="index.html">
                                <img class="logo-img logo-img-main" src="img/logo.png" alt="Asentus Logo">
                                <img class="logo-img logo-img-active" src="img/logo-dark.png" alt="Asentus Logo">
                            </a>
                        </div>
                        
                    </div>
                </div>
            </nav>
            
        </header>
        

        
        <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
            <div class="container">
                
                <ol class="carousel-indicators">
                    <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
                    <li data-target="#carousel-example-generic" data-slide-to="1"></li>
                </ol>
            </div>

            
            <div class="carousel-inner" role="listbox">
                <div class="item active">
                    <img class="img-responsive" src="img/1920x1080/01.jpg" alt="Slider Image">
                    <div class="container">
                        <div class="carousel-centered">
                            <div class="margin-b-40">
                                <h1 class="carousel-title">Hi-Tech Design</h1>
                                <p>Lorem ipsum dolor amet consectetur adipiscing dolore magna aliqua <br/> enim minim estudiat veniam siad venumus dolore</p>
                            </div>
                            <a href="#" class="btn-theme btn-theme-sm btn-white-brd text-uppercase">Explore</a>
                        </div>
                    </div>
                </div>
                <div class="item">
                    <img class="img-responsive" src="img/1920x1080/02.jpg" alt="Slider Image">
                    <div class="container">
                        <div class="carousel-centered">
                            <div class="margin-b-40">
                                <h2 class="carousel-title">Hi-Tech Design</h2>
                                <p>Lorem ipsum dolor amet consectetur adipiscing dolore magna aliqua <br/> enim minim estudiat veniam siad venumus dolore</p>
                            </div>
                            <a href="#" class="btn-theme btn-theme-sm btn-white-brd text-uppercase">Explore</a>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        

        
        
        <div class="content-lg container">
            <div class="row margin-b-40">
                <div class="col-sm-6">
                    <h2>Актуальные проекты</h2>
                    <p>Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incididunt ut laboret dolore magna aliqua enim minim veniam exercitation</p>
                </div>
            </div>
            

            <div class="row">
                
                <div class="col-sm-4 sm-margin-b-50">
                    <div class="margin-b-20">
                        <div class="wow zoomIn" data-wow-duration=".3" data-wow-delay=".1s">
                            <img class="img-responsive" src="img/970x647/01.jpg" alt="Latest Products Image">
                        </div>
                    </div>
                    <h4><a href="#">Треугольные крыши</a> <span class="text-uppercase margin-l-20">Архитектура</span></h4>
                    <p>Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incdidunt ut laboret dolor magna ut consequat siad esqudiat dolor</p>
                    <a class="link" href="#">Узнать больше</a>
                </div>
                

                
                <div class="col-sm-4 sm-margin-b-50">
                    <div class="margin-b-20">
                        <div class="wow zoomIn" data-wow-duration=".3" data-wow-delay=".1s">
                            <img class="img-responsive" src="img/970x647/02.jpg" alt="Latest Products Image">
                        </div>
                    </div>
                    <h4><a href="#">Изогнутые углы</a> <span class="text-uppercase margin-l-20">Архитектура</span></h4>
                    <p>Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incdidunt ut laboret dolor magna ut consequat siad esqudiat dolor</p>
                    <a class="link" href="#">Узнать больше</a>
                </div>
                

                
                <div class="col-sm-4 sm-margin-b-50">
                    <div class="margin-b-20">
                        <div class="wow zoomIn" data-wow-duration=".3" data-wow-delay=".1s">
                            <img class="img-responsive" src="img/970x647/03.jpg" alt="Latest Products Image">
                        </div>
                    </div>
                    <h4><a href="#">Зеленые птицы</a> <span class="text-uppercase margin-l-20">Социальное развитие</span></h4>
                    <p>Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incdidunt ut laboret dolor magna ut consequat siad esqudiat dolor</p>
                    <a class="link" href="#">Узнать больше</a>
                </div>
                
            </div>
            
        </div>
        

        
        <div class="bg-color-sky-light">
            <div class="content-lg container">
                
                <div class="swiper-slider swiper-clients">
                    
                    <div class="swiper-wrapper">
                        <div class="swiper-slide">
                            <img class="swiper-clients-img" src="img/clients/01.png" alt="Clients Logo">
                        </div>
                        <div class="swiper-slide">
                            <img class="swiper-clients-img" src="img/clients/02.png" alt="Clients Logo">
                        </div>
                        <div class="swiper-slide">
                            <img class="swiper-clients-img" src="img/clients/03.png" alt="Clients Logo">
                        </div>
                        <div class="swiper-slide">
                            <img class="swiper-clients-img" src="img/clients/04.png" alt="Clients Logo">
                        </div>
                        <div class="swiper-slide">
                            <img class="swiper-clients-img" src="img/clients/05.png" alt="Clients Logo">
                        </div>
                        <div class="swiper-slide">
                            <img class="swiper-clients-img" src="img/clients/06.png" alt="Clients Logo">
                        </div>
                    </div>
                    
                </div>
                
            </div>
        </div>
        

        
        <div class="content-lg container">
            <div class="row">
                <div class="col-sm-9">
                    <h2>Отзывы клиентов</h2>

                    
                    <div class="swiper-slider swiper-testimonials">
                        <div class="swiper-wrapper">
                            <div class="swiper-slide">
                                <blockquote class="blockquote">
                                    <div class="margin-b-20">
                                        Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incididunt ut laboret dolore magna aliqua. Ut enim minim veniam exercitation laboris ut siad consequat siad minim enum esqudiat dolore.
                                    </div>
                                    <div class="margin-b-20">
                                        Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incididunt ut laboret tempor incididunt dolore magna consequat siad minim aliqua.
                                    </div>
                                    <p><span class="fweight-700 color-link">Джон</span>, Дровосек</p>
                                </blockquote>
                            </div>
                            <div class="swiper-slide">
                                <blockquote class="blockquote">
                                    <div class="margin-b-20">
                                        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                                    </div>
                                    <div class="margin-b-20">
                                        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
                                    </div>
                                    <p><span class="fweight-700 color-link">Алекс</span>, Разработчик</p>
                                </blockquote>
                            </div>
                        </div>
                        <div class="swiper-testimonials-pagination"></div>
                    </div>
                </div>
            </div>
        </div>

        <div class="bg-color-sky-light">
            <div class="content-lg container">
                <div class="row row-space-1">
                    <div class="col-sm-4 sm-margin-b-2">
                        <div class="pricing">
                            <div class="margin-b-30">
                                <i class="pricing-icon icon-chemistry"></i>
                                <h3>Starter Kit <span> - $</span> 49</h3>
                                <p>Lorem ipsum dolor amet consectetur ut consequat siad esqudiat dolor</p>
                            </div>
                            <ul class="list-unstyled pricing-list margin-b-50">
                                <li class="pricing-list-item">Базовые функции</li>
                                <li class="pricing-list-item">До 5 проектов</li>
                                <li class="pricing-list-item">50 пользователей</li>
                            </ul>
                            <a href="" class="btn-theme btn-theme-sm btn-default-bg text-uppercase">Выбрать</a>
                        </div>
                    </div>
                    <div class="col-sm-4 sm-margin-b-2">
                        <div class="pricing pricing-active">
                            <div class="margin-b-30">
                                <i class="pricing-icon icon-badge"></i>
                                <h3>Professional <span> - $</span> 149</h3>
                                <p>Lorem ipsum dolor amet consectetur ut consequat siad esqudiat dolor</p>
                            </div>
                            <ul class="list-unstyled pricing-list margin-b-50">
                                <li class="pricing-list-item">Базовые функции</li>
                                <li class="pricing-list-item">До 100 проектов</li>
                                <li class="pricing-list-item">100 пользователей</li>
                            </ul>
                            <a href="" class="btn-theme btn-theme-sm btn-default-bg text-uppercase">Выбрать</a>
                        </div>
                    </div>
                    <div class="col-sm-4">
                        <div class="pricing">
                            <div class="margin-b-30">
                                <i class="pricing-icon icon-shield"></i>
                                <h3>Advanced <span> - $</span> 249</h3>
                                <p>Lorem ipsum dolor amet consectetur ut consequat siad esqudiat dolor</p>
                            </div>
                            <ul class="list-unstyled pricing-list margin-b-50">
                                <li class="pricing-list-item">Все функции</li>
                                <li class="pricing-list-item">Безлимитные проекты</li>
                                <li class="pricing-list-item">Безлимитные аккаунты</li>
                            </ul>
                            <a href="" class="btn-theme btn-theme-sm btn-default-bg text-uppercase">Выбрать</a>
                        </div>
                        
                    </div>
                </div>
                
            </div>
        </div>
        

        
        <div class="promo-section overflow-h">
            <div class="container">
                <div class="clearfix">
                    <div class="ver-center">
                        <div class="ver-center-aligned">
                            <div class="promo-section-col">
                                <h2>Забота о клиентах</h2>
                                <p>Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incididunt ut laboret dolore magna aliqua enim minim veniam exercitation ipsum dolor sit amet consectetur adipiscing elit sed tempor incididunt ut laboret dolore magna aliqua enim minim veniam exercitation</p>
                                <p>Ipsum dolor sit amet consectetur adipiscing elit sed tempor incididut ut sead laboret dolore magna aliqua enim minim veniam exercitation ipsum dolor sit amet consectetur adipiscing</p>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <div class="promo-section-img-right">
                <img class="img-responsive" src="img/970x970/01.jpg" alt="Content Image">
            </div>
        </div>
        

        
        <div class="bg-color-sky-light overflow-h">
            <div class="content-lg container">
                <div class="row margin-b-40">
                    <div class="col-sm-6">
                        <h2>Галерея</h2>
                        <p>Lorem ipsum dolor sit amet consectetur adipiscing elit sed tempor incididunt ut laboret dolore magna aliqua enim minim veniam exercitation</p>
                    </div>
                </div>
                

                
                <div class="masonry-grid">
                    <div class="masonry-grid-sizer col-xs-6 col-sm-6 col-md-1"></div>
                    <div class="masonry-grid-item col-xs-12 col-sm-6 col-md-8">
                        
                        <div class="work wow fadeInUp" data-wow-duration=".3" data-wow-delay=".1s">
                            <div class="work-overlay">
                                <img class="full-width img-responsive" src="img/800x400/01.jpg" alt="Portfolio Image">
                            </div>
                            <div class="work-content">
                                <h3 class="color-white margin-b-5">Исскуство архитектуры</h3>
                                <p class="color-white margin-b-0">Lorem ipsum dolor sit amet consectetur adipiscing</p>
                            </div>
                            <a class="content-wrapper-link" href="#"></a>
                        </div>
                        
                    </div>
                    <div class="masonry-grid-item col-xs-6 col-sm-6 col-md-4">
                        
                        <div class="work wow fadeInUp" data-wow-duration=".3" data-wow-delay=".2s">
                            <div class="work-overlay">
                                <img class="full-width img-responsive" src="img/397x400/01.jpg" alt="Portfolio Image">
                            </div>
                            <div class="work-content">
                                <h3 class="color-white margin-b-5">Эстетичный дизайн</h3>
                                <p class="color-white margin-b-0">Lorem ipsum dolor sit amet consectetur adipiscing</p>
                            </div>
                            <a class="content-wrapper-link" href="#"></a>
                        </div>
                        
                    </div>
                    <div class="masonry-grid-item col-xs-6 col-sm-6 col-md-4">
                        
                        <div class="work wow fadeInUp" data-wow-duration=".3" data-wow-delay=".3s">
                            <div class="work-overlay">
                                <img class="full-width img-responsive" src="img/397x300/01.jpg" alt="Portfolio Image">
                            </div>
                            <div class="work-content">
                                <h3 class="color-white margin-b-5">Эстетичный дизайн</h3>
                                <p class="color-white margin-b-0">Lorem ipsum dolor sit amet consectetur adipiscing</p>
                            </div>
                            <a class="content-wrapper-link" href="#"></a>
                        </div>
                        
                    </div>
                    <div class="masonry-grid-item col-xs-6 col-sm-6 col-md-4">
                        
                        <div class="work wow fadeInUp" data-wow-duration=".3" data-wow-delay=".4s">
                            <div class="work-overlay">
                                <img class="full-width img-responsive" src="img/397x300/02.jpg" alt="Portfolio Image">
                            </div>
                            <div class="work-content">
                                <h3 class="color-white margin-b-5">Эстетичный дизайн</h3>
                                <p class="color-white margin-b-0">Lorem ipsum dolor sit amet consectetur adipiscing</p>
                            </div>
                            <a class="content-wrapper-link" href="#"></a>
                        </div>
                        
                    </div>
                    <div class="masonry-grid-item col-xs-6 col-sm-6 col-md-4">
                        
                        <div class="work wow fadeInUp" data-wow-duration=".3" data-wow-delay=".5s">
                            <div class="work-overlay">
                                <img class="full-width img-responsive" src="img/397x300/03.jpg" alt="Portfolio Image">
                            </div>
                            <div class="work-content">
                                <h3 class="color-white margin-b-5">Эстетичный дизайн</h3>
                                <p class="color-white margin-b-0">Lorem ipsum dolor sit amet consectetur adipiscing</p>
                            </div>
                            <a class="content-wrapper-link" href="#"></a>
                        </div>
                        
                    </div>
                </div>
                
            </div>
        </div>
        
        

        
        <footer class="footer">
            
            <div class="footer-seperator">
                <div class="content-lg container">
                    <div class="row">
                        <div class="col-sm-4 sm-margin-b-30">
                            
                            <ul class="list-unstyled footer-list">
                                <li class="footer-list-item"><a class="footer-list-link" href="#">Twitter</a></li>
                                <li class="footer-list-item"><a class="footer-list-link" href="#">Facebook</a></li>
                                <li class="footer-list-item"><a class="footer-list-link" href="#">Instagram</a></li>
                                <li class="footer-list-item"><a class="footer-list-link" href="#">YouTube</a></li>
                            </ul>
                            
                        </div>
                        <div class="col-sm-5 sm-margin-b-30">
                            <h2 class="color-white">Напишите нам</h2>
                            <input type="text" class="form-control footer-input margin-b-20" placeholder="Имя" required>
                            <input type="email" class="form-control footer-input margin-b-20" placeholder="Почта" required>
                            <input type="text" class="form-control footer-input margin-b-20" placeholder="Телефон" required>
                            <textarea class="form-control footer-input margin-b-30" rows="6" placeholder="Сообщение" required></textarea>
                            <button type="submit" class="btn-theme btn-theme-sm btn-base-bg text-uppercase">Отправить</button>
                        </div>
                    </div>
                    
                </div>
            </div>
            

            
            <div class="content container">
                <div class="row">
                    <div class="col-xs-6">
                        <img class="footer-logo" src="img/logo.png" alt="Asentus Logo">
                    </div>
                    <div class="col-xs-6 text-right">
                        <p class="margin-b-0"><a class="color-base fweight-700" href="">HI-TECH</a> -  Разработано с любовью, <a class="color-base fweight-700" href="">Code Union</a></p>
                    </div>
                </div>
                
            </div>
            
        </footer>
        

        
        <a href="javascript:void(0);" class="js-back-to-top back-to-top">Top</a>

        
        
        <script src="vendor/jquery.min.js" type="text/javascript"></script>
        <script src="vendor/jquery-migrate.min.js" type="text/javascript"></script>
        <script src="vendor/bootstrap/js/bootstrap.min.js" type="text/javascript"></script>

        
        <script src="vendor/jquery.easing.js" type="text/javascript"></script>
        <script src="vendor/jquery.back-to-top.js" type="text/javascript"></script>
        <script src="vendor/jquery.smooth-scroll.js" type="text/javascript"></script>
        <script src="vendor/jquery.wow.min.js" type="text/javascript"></script>
        <script src="vendor/swiper/js/swiper.jquery.min.js" type="text/javascript"></script>
        <script src="vendor/masonry/jquery.masonry.pkgd.min.js" type="text/javascript"></script>
        <script src="vendor/masonry/imagesloaded.pkgd.min.js" type="text/javascript"></script>

        
        <script src="js/layout.min.js" type="text/javascript"></script>
        <script src="js/components/wow.min.js" type="text/javascript"></script>
        <script src="js/components/swiper.min.js" type="text/javascript"></script>
        <script src="js/components/masonry.min.js" type="text/javascript"></script>

    </body>
    
</html>
