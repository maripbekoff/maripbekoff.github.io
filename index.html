<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>#maripbekoff</title>
	<link rel="stylesheet" type="text/css" media="screen" href="styles/style.css">
	<link rel="icon" href="favicon.ico">
</head>
<body>
	<div id="particles-js"></div>

	<div class="wrapper">
		<div class="bgimg">
			<a href="https://vk.com/chingizsck">
				<div class="middle">
					<h1>COMING SOON</h1>
					<hr>
					<p id="demo">10 mins</p>
				</div>
			</a>
			<div class="bottomleft">
				<p>#maripbekoff</p>
			</div>
		</div>
	</div>


	<script src="particles.js"></script>
	<script type="text/javascript">
		particlesJS.load('particles-js', 'particles.json', function() {
  		console.log('callback - particles.js config loaded');
		});
	</script>

	<script type="text/javascript">
// Set the date we're counting down to
var countDownDate = new Date("March 31, 2019 00:00:00").getTime();

// Update the count down every 1 second
var x = setInterval(function() {

  // Get todays date and time
  var now = new Date().getTime();

  // Find the distance between now an the count down date
  var distance = countDownDate - now;

  // Time calculations for days, hours, minutes and seconds
  var days = Math.floor(distance / (1000 * 60 * 60 * 24));
  var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
  var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
  var seconds = Math.floor((distance % (1000 * 60)) / 1000);

  // Display the result in an element with id="demo"
  document.getElementById("demo").innerHTML = days + "d " + hours + "h "
  + minutes + "m " + seconds + "s ";

  // If the count down is finished, write some text
  if (distance < 0) {
    clearInterval(x);
    document.getElementById("demo").innerHTML = "EXPIRED";
    window.location.replace("Direct/index.html");
  }
}, 1000); 
	</script>
</body>
</html>
