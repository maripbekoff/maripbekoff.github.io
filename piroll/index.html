<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <meta name="author" content="code union">
    <meta name="describe" content="#maripbekoff">
    <title>Piroll | Welcome!</title>
    <link rel="stylesheet" href="path/to/pe-icon-7-stroke/css/pe-icon-7-stroke.css">
    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="./styles/style.css">
</head>

<body>
    <header>
        <div class="logo">
            <img src="./img/logo.png" alt="logo">
        </div>
        <nav>
            <ul>
                <li><a href="index.html" class="active">home</a></li>
                <li><a href="#">about</a></li>
                <li><a href="#">contact</a></li>
            </ul>
        </nav>
    </header>
    <div class="clearfix"></div>
    <section>
        <div class="title">
            <h1>UI/UX & Graphic Designer</h1>
            <p>
                I am a Graphic & Web Designer based in New York, specializing<br>in User Interface Design and
                Development.
            </p>
        </div>
    </section>
    <section class="works">
        <div class="items dark_gray"></div>
        <div class="items gray"></div>
        <div class="items dark_gray"></div>
        <div class="items gray"></div>
        <div class="items gray"></div>
        <div class="items green"><i class="pe-7s-look"></i></div>
        <div class="items gray"></div>
        <div class="items dark_gray"></div>
        <div class="items dark_gray"></div>
        <div class="items gray"></div>
        <div class="items dark_gray"></div>
        <div class="items gray"></div>
    </section>
    <div class="footer">
        <div class="wrapper">
            <div class="about">
                <h3>Code Union, Inc.</h3>
                <p>&copy; 2019 Code Union. All rights reserved.<br>Designed by #maripbekoff.</p>
            </div>
            <div class="mail">
                <ul>
                    <li><a href="mailto:tech.codeunion@gmail.com">tech.codeunion@gmail.com</a></li>
                    <li>+7 777 418 3462</li>
                </ul>
            </div>
            <div class="navigation">
                <ul>
                    <li><a href="#">Projects</a></li>
                    <li><a href="#">About</a></li>
                    <li><a href="#">Services</a></li>
                    <li><a href="#">Carreer</a></li>
                </ul>
            </div>
            <div class="more">
                <ul>
                    <li><a href="#">News</a></li>
                    <li><a href="#">Events</a></li>
                    <li><a href="#">Contact</a></li>
                    <li><a href="#">Legals</a></li>
                </ul>
            </div>
            <div class="social">
                <ul>
                    <li><a href="#">Facebook</a></li>
                    <li><a href="#">Twitter</a></li>
                    <li><a href="#">Instagram</a></li>
                    <li><a href="#">Dribbble</a></li>
                </ul>
            </div>
        </div>
    </div>
</body>

</html>