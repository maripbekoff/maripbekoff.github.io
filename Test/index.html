<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>title</title>
	<link rel="stylesheet" type="text/css" href="styles/style.css">
</head>
<body>
	<div id="particles-js"></div>

	<script src="particles.js"></script>
	<script type="text/javascript">
		particlesJS.load('particles-js', 'particles.json', function() {
  		console.log('callback - particles.js config loaded');
		});
	</script>
</body>
</html>