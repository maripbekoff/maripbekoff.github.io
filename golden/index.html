<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <meta name="author" content="Code Union #maripbekoff">
    <meta name="describe" content="#maripbekoff">
    <title>Golden | Welcome!</title>
    <link rel="stylesheet" href="./styles/style.css">
</head>

<body>
    <header>
        <div class="container">
            <div class="wrapper clearfix">
                <img src="./img/logo.png" alt="golden logo" class="logo">
                <nav>
                    <ul class="menu">
                        <li><a href="index.html">home</a></li>
                        <li><a href="#services">services</a></li>
                        <li><a href="#works">portfolio</a></li>
                        <li><a href="#about">about</a></li>
                        <li><a href="#contact">contact</a></li>
                    </ul>
                </nav>
            </div>
            <div class="heading">
                <div class="heading_first">
                    Welcome to our site!
                </div>
                <h1>It's nice to meet you</h1>
            </div>
            <a class="button" href="https://vk.com/chingizsck" target="_blank">tell me more</a>
        </div>
    </header>
    <section id="services">
        <div class="container">
            <div class="title">
                <h2>services</h2>
                <p>Lorem ipsum dolor sit, amet consectetur adipisicing.</p>
            </div>
            <div class="grid">
                <div class="commerce inner">
                    <img src="./img/icon1.png" alt="icon1">
                    <h3>E-commerce</h3>
                    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Soluta unde, corrupti ut aperiam
                        consequatur distinctio alias ex provident impedit, praesentium beatae architecto at ratione
                        possimus mollitia blanditiis voluptate maxime nesciunt.</p>
                </div>
                <div class="web inner">
                    <img src="./img/icon2.png" alt="icon2">
                    <h3>Responsive Web</h3>
                    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Magnam illo quia accusamus quibusdam
                        architecto cumque iure doloribus dolor recusandae quas dicta deleniti, perspiciatis porro hic
                        harum id? Minus, dicta recusandae.</p>
                </div>
                <div class="security inner">
                    <img src="./img/icon3.png" alt="icon3">
                    <h3>Web Security</h3>
                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quam, consequatur? Ducimus ipsa
                        repellat, error ipsum temporibus adipisci, laudantium quam sed earum placeat distinctio aperiam
                        consequatur ad molestias architecto, vel eaque?</p>
                </div>
            </div>
        </div>
    </section>
    <section id="works">
        <div class="container">
            <div class="title">
                <h2>our portfolio</h2>
                <p>Lorem ipsum dolor sit amet consectetur adipisicing.</p>
            </div>
            <div class="grid">
                <div class="img1">
                    <img src="./img/img1.jpg" alt="img">
                </div>
                <div class="img2">
                    <img src="./img/img2.jpg" alt="img">
                </div>
                <div class="img3">
                    <img src="./img/img3.jpg" alt="img">
                </div>
            </div>
        </div>
    </section>
    <section id="about">
        <div class="container">
            <div class="title">
                <h2>about us</h2>
                <p>Lorem ipsum dolor sit amet consectetur adipisicing.</p>
            </div>
            <div class="about">
                <div class="beginning">
                    <h3>july 2010<br>Our Humble Beginnings</h3>
                    <p>
                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nesciunt at dolor quia sequi molestias
                        iusto recusandae dicta. Itaque est dolorum vitae, inventore vel ad obcaecati. Placeat cupiditate
                        corporis dolores obcaecati!
                    </p>
                </div>
                <div class="img_begin">
                    <img src="./img/about1.jpg" alt="">
                </div>
                <div class="cls"></div>
                <div class="cls"></div>
                <div class="img_startup">
                    <img src="./img/about2.jpg" alt="">
                </div>
                <div class="startup">
                    <h3>january 2011<br>Facing Startup Battles</h3>
                    <p>
                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque ab animi voluptatibus
                        veritatis beatae illum eligendi ea molestiae vel architecto dolorum repudiandae, dolorem earum
                        debitis est, soluta quidem, nobis possimus.
                    </p>
                </div>
                <div class="dark_days">
                    <h3>December 2012<br>Enter the Dark Days</h3>
                    <p>
                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora, cupiditate dolor. Atque
                        dolorem
                        voluptates facilis natus fugiat unde doloremque, laudantium voluptas veritatis repellat ipsam
                        sapiente, asperiores corrupti quibusdam ea enim.
                    </p>
                </div>
                <div class="img_darkd">
                    <img src="./img/about3.jpg" alt="">
                </div>
                <div class="cls"></div>
                <div class="cls"></div>
                <div class="img_triumph">
                    <img src="./img/about4.jpg" alt="">
                </div>
                <div class="triumph">
                    <h3>February 2014<br>Our Triumph</h3>
                    <p>
                        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Beatae, eaque amet harum distinctio
                        illum cupiditate rerum temporibus. Maiores vero provident, ad excepturi odio sequi, nisi
                        deserunt quidem natus voluptatum velit?
                    </p>
                </div>
            </div>
        </div>
    </section>
    <footer id="contact">
        <div class="container">
            <p>&copy;Copyright 2019 Code Union</p>
        </div>
    </footer>
</body>

</html>